docs/modules/plugins/pages/plugin-excel.adoc

```diff
@@ -128,6 +128,30 @@ When I create temporary excel file containing sheet with name `movies` and conte
 and put its path to scenario variable `movies-path`
 ----
 
+=== Add sheet to existing excel file
+
+Add new sheet with the specified name and content to the existing excel file.
```

Comment:
What will happen if the sheet already exists?

Reply:
discussed